src/main/java/org/opentripplanner/graph_builder/module/AddTransitModelEntitiesToGraph.java

```diff
@@ -277,8 +277,8 @@ private void createElevatorEdgesAndAddThemToGraph(
             toVertexLevelName
         );
 
-        new PathwayEdge(fromVertex, fromOffboardVertex, fromVertex.getName());
-        new PathwayEdge(toOffboardVertex, toVertex, toVertex.getName());
+        PathwayEdge.withComputedDistance(fromVertex, fromOffboardVertex, fromVertex.getName());
+        PathwayEdge.withComputedDistance(toOffboardVertex, toVertex, toVertex.getName());
```

Comment:
This creates edges for elevators, which means that the length _could be_ 0, if both coordinates are the same. 

Looking at the code, it seems that in the case of elevators the `PathwayEdge` is merely for linking the pathway vertices with the elevator vertices. Because of this a 0 traversal time is expected for the `PathwayEdge` -- the `ElevatorHopEdge` makes use of the `traversal_time` or uses the `elevatorHopCost` / `elevatorHopTime` parameters.

If both the  `length` and `time` is `<= 0` then the `PathwayEdge` could have a `time = 0` and `cost = 1`, which would be similar to the `FreeEdge` usage for OSM based elevators.